### DISCUSSION TITLE
I don't quite get GitHub Organizations

### DISCUSSION THREAD

**Discussion by rfalanga** [community/community#60364] [Repositories] [answered]
### Select Topic Area

Question

### Body

I've been using GitHub (GH), on and off, for 5 years. In all that time I've never created an organization. If any of my repos are in an organization, then they're in the default organization, whatever that's called.

I've read about GH Organizations, but I still don't get it. I've got experience with an old version of TFS (TFS 2015). It uses "Collections". I know that Azure DevOps Services and Server both use Organizations. So, they are analogous. But there are differences, which I'm wondering if GH Organizations would be better suited for. For example, with TFS and Azure DevOps their Projects can have multiple repos in them. GH doesn't have the same idea in its projects. 

I'm involved in migrating lots of TFS/TFVC repos to GH. This is a major project. And there are sometimes conflicts because someone who creates a TFVC repo using the name **Develop** and someone else would create a different TFVC repo in a different TFS Project, also named **Develop**. No problem, with TFS, because they're different TFS projects. But there's a problem with GH, because you cannot have two repos both named **Develop** in the same organization.

So, is that what organizations are for??

And another question, how do organizations work with personal GH accounts? I've got a GH Professional account, and I know I can create organizations, but I'm stumped to know how many I can create, how many repos I can put into any organization, etc.

**Comment by ronakjkothari** [comment, marked as answer]
To put it simply, GitHub organizations are used to organize a group of members and repositories under one singular organization. I use it to organize separate repositories that are still part of the same company under one organization. There are many other advantages of using GH Organizations.

1. Collaborative Development: GitHub Organizations provide a centralized platform for collaboration, allowing multiple individuals to work together on projects and enabling better coordination, code sharing, and streamlined workflows, more than what would be offered with an individual repository.
2. Permissions Control: Organizations offer granular access control and permissions management. You can assign different levels of access to members, controlling who can view, clone, push, or merge changes to repositories. This ensures that only authorized individuals can contribute to specific projects and helps maintain the security and integrity of your codebase.
3. Team Management: Organizations allow you to create teams within the organization. Teams can be structured based on departments, roles, or specific projects. This builds off of the above point well.
4. Enhanced Visibility and Branding: GitHub Organizations provide a dedicated space where you can showcase your organization's projects, repositories, and contributions. It offers better visibility to your work and allows you to establish a professional presence. Additionally, organizations can have their branding, logos, and descriptions.
5. Scalability: Organizations are designed to accommodate growth and scalability. As your team or project expands, you can easily add or remove members, create new repositories, and manage access permissions.

As for the numbers, you can have an unlimited number of repositories and members per organization. I don't think there is a limit on the number of organizations an individual can create.

**Reply by rfalanga** [reply]
Thank you, @ronakkothari123, I really like your answer. You gave several reasons and included what the motivating factor was. I can now see how organizations can address a need, such as improved permissions control, organizing people around projects, etc.

Thank you, again.

**Reply by ronakjkothari** [reply]
No problem! 😄

**Reply by gooddavvy** [reply]
Whoa, I wanted to make an answer for this @ronakkothari123! That was **awesome**!!!

**Reply by kalin3ar** [reply]
That makes sense, but what I don't understand is why I'm not allowed to create any organizations when on the Enterprise plan (Trial period). I get booted back to the pricing page when I attempt. The only other option I see is to convert my username into an organization, which isn't what I had in mind. And the pricing page says everything should be included in the Enterprise plan...yet Organizations do not seem to be.

**Reply by ronakjkothari** [reply]
Enterprise accounts enable users to centrally control their multiple organizations. My guess is that you aren't looking for the benefits that enterprise plans offer. I would read the [GitHub pricing](https://github.com/pricing) for more details, however, from what I see, you should still be able to create and remove organizations.

Sorry I wasn't able to be as helpful.
